consensus/replay_file.go

```diff
@@ -41,7 +42,11 @@ func (cs *ConsensusState) ReplayFile(file string, console bool) error {
 	cs.startForReplay()
 
 	// ensure all new step events are regenerated as expected
-	newStepCh := subscribeToEvent(cs.evsw, "replay-test", types.EventStringNewRoundStep(), 1)
+	newStepCh := make(chan interface{})
```

Comment:
if tests pass without it, does it mean it is not needed or it is just we do not cover this in tests?